HelpSource/Classes/ServerMeter.schelp

```diff
@@ -42,13 +42,33 @@ note::
 A ServerMeter encapsulates both a link::Classes/Window:: and a link::Classes/ServerMeterView:: within that Window. For more information about Windows and views see link::Guides/GUI-Introduction#Basic elements: Windows, views and containers::
 ::
 
+METHOD:: position
+
+the position of the ServerMeter window.
+The value should be a link::Classes/Point::, e.g. teletype::x@y::, or respond to teletype::.asPoint::.
+
 
 EXAMPLES::
 
 code::
-
 s = Server.internal; // use the internal server
 s.boot;
+
 // display 4 input channels & main stereo output
 m = ServerMeter.new(s, 4, 2);
+
+// Query the current window position
+m.position //-> Position(left:5.0, bottom:277.0)
```

Comment:
see my comment below w.r.t. a position getter method.

Reply:
Thanks, done in https://github.com/supercollider/supercollider/pull/6283/commits/4ad70d94f4ff8b4ee959c3dd6c9c60061800dc7c!